src/robusta/core/persistency/scheduled_jobs_states_dal.py

```diff
@@ -21,9 +21,13 @@ def __init__(self):
         self.__init_scheduler_dal()
 
     def __load_config_map(self) -> ConfigMap:
-        return ConfigMap.readNamespacedConfigMap(
-            JOBS_CONFIGMAP_NAME, CONFIGMAP_NAMESPACE
-        ).obj
+        try:
```

Comment:
This breaks the logic here
The scheduler config map is created lazy.
See SchedulerDal.__init_scheduler_dal